src/Rules/Rules.csproj

```diff
@@ -35,4 +35,79 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Are the changes in rules.csproj necessary? It seems like things were working without them before...

Reply:
They weren't causing any problems before because the strings in the resx file hadn't changed since the conversion to .Net Standard. Now that the strings have been updated, if I didn't make this change, the *.designer.cs files wouldn't be updated. And thus the code wouldn't compile.

Reply:
We also have resx files in Actions, Automation, Core, Desktop, and RuleSelection. Are those projects currently in a state where changes wouldn't get picked up?

Reply:
I will check after this PR.